CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 1.1.3
```

Comment:
Considering there are new features it's at least `1.2` (rigorously speaking it should even be `2.0` because of breaking change :stuck_out_tongue: ) 
```suggestion
# 1.2
```

Reply:
Agreed that it should have been `1.2`, but let's keep `1.1.3` for this one. I'll be more careful next time

Reply:
I don't get why waiting to be careful, it's not released yet so we can still do the right thing :)